src/components/HURUmap/Chart/utils.js

```diff
@@ -0,0 +1,31 @@
+export const slugify = (string) => {
```

Comment:
[PT](https://github.com/CodeForAfrica/PromiseTracker/blob/e70c0928046110fd5104be807d3bd410ba59d93d/src/utils.js#L1-L18) `slugify` seem to handle even special characters.